src/DDTrace/Propagators/TextMap.php

```diff
@@ -67,6 +70,7 @@ public function extract($carrier)
 
         $spanContext = new SpanContext($traceId, $spanId, null, $baggageItems, true);
         $this->extractPrioritySampling($spanContext, $carrier);
+        $this->extractOrigin($spanContext, $carrier);
```

Comment:
extracting the origin is only valid if we were able to successfully extract a trace id.

Reply:
Thanks for clarifying! This line only gets run in this case as [on line 67-69](https://github.com/DataDog/dd-trace-php/pull/615/files#diff-bca28826d2bcd463c28ea6562b776fccR67-R69) there is a null-check on the trace ID. :)